### PR TITLE
Travis

### LINKED ISSUE #4: Travis CI
We need Travis or some other CI system setup. This should at least run `coala` and should also run the required tests.

### PR DESCRIPTION
**Pull request by frextrite**
Add Travis CI

Closes #4

### CODE REVIEW

**Comment by ksdme** [comment]
If you think adding `coala` requires time, feel free to make another issue regarding adding `coala` to Travis config and merge this PR with build tests.

File: travis.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (ksdme) [inline travis.sh]: Please move this file to more appropriate location. `.ci/travis.sh`?

**Comment by ksdme** [comment]
It looks like you have tests as well, I don't see them being run here.

File: travis.sh (hunk 1 above)
Comment (frextrite) [reply travis.sh]: That'll be better. I'll move it.

**Comment by frextrite** [comment]
> It looks like you have tests on the repo too, I don't see them being run here.

I ran the script locally first and it was running all the tests fine. I'll take a look again.

**Comment by frextrite** [comment]
Closes #4

**Pull request merged**